application/src/main/resources/application.yml

```diff
@@ -10,7 +10,6 @@ hono:
     singleTenant: false
     insecurePortEnabled: false
     bindAddressInsecurePort: 127.0.0.1
```

Comment:
Actually, we do not need to configure them. When the insecure port was still contained here, it was a convenient way to have all values together.
Now it seems better to completely remove them - WDYT?